### ISSUE TITLE
 Installation Fails (unable to open wpcap.lib) if using Python 3.11

### ISSUE THREAD

**Issue by jc-bm** [bnjmnp/pysoem#93]
Pretty much identical to #48 incremented 1, I can install with 3.10 but not with 3.11. Does the wheel need to be rebuilt with every version increment? If so, it could be helpful to specify the upper bound on Python.

**Comment by bnjmnp** [comment]
Right, there is no wheel for Python 3.11 for now. The available wheels you can see on PyPI: [https://pypi.org/project/pysoem/#files](https://pypi.org/project/pysoem/#files)

I should maybe also work on somehow putting the missing `wpcap.lib` in the "Source Distribution". With that one could build pysoem without the wheel right away, also on Windows.

As you have the Visual Studio compiler installed you could also try to install pysoem from the GitHub sources: `pip install git+https://github.com/bnjmnp/pysoem.git` (see: [https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-from-vcs](https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-from-vcs))
